server/test/lib/actions/fusor/deployment/rhev/deploy_test.rb

```diff
@@ -7,7 +7,7 @@ class DeployTest < FusorActionTest
     CreateEngineHostRecord = ::Actions::Fusor::Deployment::Rhev::CreateEngineHostRecord
 
     def setup
-      @deploy = create_action Deploy
+      @deploy = create_action Actions::Fusor::Deployment::Rhev::Deploy
```

Comment:
Not sure why this was suddenly causing problems, but it was unable to resolve Deploy with the fully qualified name.